includes/Story_Post_Type.php

```diff
@@ -465,6 +465,13 @@ public function get_editor_settings() {
 				 * Creation date: 2020-06-30
 				 */
 				'rowBasedGallery'              => false,
+				/**
+				 * Description: Flag for using an as-you-type search in media and media3p with a debouncer.
+				 * Author: @diegovar
+				 * Issue: #3206
+				 * Creation date: 2020-07-15
+				 */
+				'autoSearchDebounceMedia'      => false,
```

Comment:
Shouldn't this be `incrementalSearchDebounceMedia`? 🤔

Reply:
yes indeed!